utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -525,8 +525,9 @@ def create_argument_parser():
     option('--stdlib-deployment-targets', store,
            type=argparse.ShellSplitType(),
            default=None,
-           help='list of targets to compile or cross-compile the Swift '
-                'standard library for. %(default)s by default.')
+           help='The targets to compile or cross-compile the Swift standard '
+                'library for. %(default)s by default. Comma separated list: '+
+           " ".join(StdlibDeploymentTarget.get_target_names()))
```

Comment:
Please use single quotes for string literals to keep the style consistent. And avoid string concatenation in this case:

```Python
help='The targets to compile or cross-compile the Swift standard '
     'library for. %(default)s by default. Comma separated list: '
     ' '.join(StdlibDeploymentTarget.get_target_names()))
```

Python has a nifty feature that automatically concatenates string literals that appear sequentially with only whitespace between them:

```Python
'Hello, ' 'world!'  # => 'Hello, world!'
```

Reply:
I need a different string literal for the join. Is this ok?
```
           help='The targets to compile or cross-compile the Swift standard '
                'library for. %(default)s by default. Comma separated list: '
           + ' '.join(StdlibDeploymentTarget.get_target_names()))
```